client/src/components/PostContainer.tsx

```diff
@@ -103,7 +118,8 @@ const PostContainer : React.FC<Props> = ({ path }: Props) => {
           )) : (
             <Empty
               image={Empty.PRESENTED_IMAGE_SIMPLE}
-              description="There are no posts yet, You need to follow users to view their posts"
+              description={path === 'bookmarks' ? "You don't have any Bookmarked Posts"
+                : 'There are no posts yet, You need to follow users to view their posts'}
```

Comment:
it can be like this which is more general for most cases 
` description={path === 'feed'? 'There are no posts yet, You need to follow users to view their posts!'
                : 'There are no posts yet!'}`